package.json

```diff
@@ -24,14 +24,6 @@
     "test:watch": "npm run test -- --watchAll"
   },
   "dependencies": {
-    "@storybook/addon-actions": "^4.1.6",
```

Comment:
lets talk about this change. I think when we minify and distribute this UI package having all these dependencies here is not necessary.  
When we import this UI package into a project as `doctors-service` we dont really need any storybook there.
I think this way the bundle will be smaller therefore the loadings faster. 
Let me know your thoughts @herman-rogers, i might be wrong.

Reply:
happy to optimize the builds so that we only have the lib and no story book in them.